.changeset/green-eels-brush.md

```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/design-system-documentation': patch
+---
+
+Removed all breaking changes tags in the migration guide.
```

Comment:
```suggestion
Cleaned up the migration guide by removing redundant "breaking" tags in the "Migration from v9 to v10" section, since all listed changes are breaking.
```